### DISCUSSION TITLE
RandomUtil.randomEleList

### DISCUSSION THREAD

**Discussion by BIGWGY** [looly/hutool#1787] [General]
![image](https://user-images.githubusercontent.com/13144389/130753760-993d9f98-1573-4a5d-90a7-7d3e3612ba3e.png)

如果我通过返回的list,   去删除 source 里面的元素,  用 list.foreach( e ->  source.remove(e) ) 就会报异常. 如果是 new ArrayList(source) 就不会了.

**Comment by looly** [comment]
嗯，确实是。

5.7.10修改此处逻辑~~

**Comment by looly** [comment]
下次在issue中提吧~~
